src/app.js

```diff
@@ -38,6 +38,12 @@ const learnerServiceLocalBaseUrl = process.env.sunbird_learner_service_local_bas
 
 const whiteListedChannelList = process.env.sunbird_content_service_whitelisted_channels
 const blackListedChannelList = process.env.sunbird_content_service_blacklisted_channels
+const whitelistedframeworkList = process.env.sunbird_content_service_whitelisted_framework
+const blacklistedframeworkList = process.env.sunbird_content_service_blacklisted_framework
+const whitelistedLanguageList = process.env.sunbird_content_service_whitelisted_language
```

Comment:
Based on design we should be considering `contentType` and `mimeType` instead of language and creator.

Reply:
@kochhar The changes are done on different branch, and i tried to push to this current branch. So this push has outdated code. I will merge the changes and push the code.

Reply:
updated